packages/fields/src/fields/title/index.ts

```diff
@@ -9,15 +9,15 @@ import { __ } from '@wordpress/i18n';
  */
 import type { BasePost } from '../../types';
 import { getItemTitle } from '../../actions/utils';
-import TitleView from './title-view';
+import { PostTitleView } from './view';
 
 const titleField: Field< BasePost > = {
 	type: 'text',
 	id: 'title',
 	label: __( 'Title' ),
 	placeholder: __( 'No title' ),
 	getValue: ( { item } ) => getItemTitle( item ),
-	render: TitleView,
+	render: PostTitleView,
```

Comment:
Shouldn't this be `PagesTitleView`? 

It seems only the pages can be set as homepage/posts, so the posts don't need to query the site for that info — they should have their own title renderer.

Reply:
If we agree about using different fields, I want to do that here (see [comment](https://github.com/WordPress/gutenberg/pull/67449#discussion_r1865498062) and PR's description).